### PR TITLE
 Blog about how to send log4J logs to EFK stack in RHOCP

### PR DESCRIPTION
**Pull request by Yushan-Lin**
https://github.ibm.com/was-liberty/WS-CD-Open/issues/19958

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
Fixes #349

File: drafts/LOG4J-RHOCP.adoc [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+{"type":"liberty_message","host":"192.168.0.104","ibm_userDir":"\/Users\/yushan.lin@ibm.com\/Documents\/archived-guide-log4j\/finish\/target\/liberty\/wlp\/usr\/","ibm_serverName":"log4j.sampleServer","message":"hello liberty servlet warning message!","ibm_threadId":"00000035","ibm_datetime":"2020-02-13T11:27:07.792-0500","module":"application.servlet.LibertyServlet","loglevel":"WARNING","ibm_methodName":"doGet","ibm_className":"application.servlet.LibertyServlet","ibm_sequence":"1581611227792_0000000000017","ext_thread":"Default Executor-thread-8"}
+```
+
+As shown, there are different options and methods to send your Log4j 2 and other non-JUL logs to the OpenShift EFK stack, based on your needs.
+
+Other related links:
```
Comment (Charlotte-Holt) [inline drafts/LOG4J-RHOCP.adoc:227]: Are these related links for the entire post? If so, I'd update to:

>**Related links**
>
>* link:https://openliberty.io/docs/ref/general/#logging.html[Open Liberty logging documentation]
>
>* link:https://developer.ibm.com/videos/use-json-logging-in-open-liberty[Use JSON logging in Open Liberty]
>
>* link:https://openliberty.io/blog/2019/12/03/custom-fields-json-logs.html[Adding custom fields to JSON logs in Open Liberty]
>
>* https://community.ibm.com/community/user/imwuc/viewdocument/on-demand-using-liberty-with-elast[Using Liberty with Elastic Stack (aka ELK)]
